operations/system_management_health/Grafana_Dashboards_by_Component.md

```diff
@@ -0,0 +1,84 @@
+# Grafana Dashboards by Component
+## _This document show dashboards available per component in Grafana_
+
+
+A Grafana dashboard is a powerful open source analytical and visualization tool that consists of multiple individual panels arranged in a grid. The panels interact with configured data sources, including.
+
+- AWS CloudWatch
+- Prometheus
+
+_NOTE: There are some known issues around some of the grafana dashboards, and those are explained in [Troubleshoot_Grafana_Dashboard.md](./Troubleshoot_Grafana_Dashboard.md)._
+## Dashboards
+
+##### Ceph Dashboards
+
+- Ceph - Cluster
```

Comment:
Line with Dashboard, since this list is already for Ceph, no need to include it on each item
```suggestion
- Cluster
```